src/main/charts/confluence/templates/_fluentd_templates.tpl

```diff
@@ -3,6 +3,9 @@
 - name: fluentd
   image: {{ .Values.fluentd.imageName }}
   command: ["fluentd", "-c", "/fluentd/etc/fluent.conf", "-v"]
+  ports:
+    - containerPort: {{ .Values.fluentd.httpPort }}
+      protocol: TCP
```

Comment:
Note that pod containers don't *need* to declare their ports, but it's good practice to do so.